src/core/server/saved_objects/export/get_sorted_objects_for_export.ts

```diff
@@ -116,8 +116,11 @@ async function fetchObjectsToExport({
     }
 
     // sorts server-side by _id, since it's only available in fielddata
-    return findResponse.saved_objects.sort((a: SavedObject, b: SavedObject) =>
-      a.id > b.id ? 1 : -1
+    return (
+      findResponse.saved_objects
+        // exclude the find-specific `score` property from the exported objects
+        .map(({ score, ...obj }) => obj)
+        .sort((a: SavedObject, b: SavedObject) => (a.id > b.id ? 1 : -1))
```

Comment:
I think this was the only significant API where we need to remove `score` after performing a `find` request.